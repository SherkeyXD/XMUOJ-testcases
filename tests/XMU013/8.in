2075321867603005310630608030707233433613846800063513004306083567467325260216877501840073651146606660083238062088833241333073171741513030812000067622030313406104803260063361780060377636782733018003338600770132714220800300
1663407860144207073064836356813806366047803607063050601400703335466550281400443100561606043361017520063633508331336106310873666171075001406438654010606057043630013688508440571077040277645372033032632100107318306463417372733653761051040653203373701066230034670222800620061035600300050707004607250087032628706007026112813684338778063352075706004031006600850168370631480080377817006803201180706006538050661605000600626205
