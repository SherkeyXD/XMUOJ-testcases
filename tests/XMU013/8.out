1663407860144207073064836356813806366047803607063050601400703335466550281400443100561606043361017520063633508331336106310873666171075001406438654010606057043630013688508440571077040277645372033032634175429185909468728003341684468284474267050173764579234340753790267945321252477801890780655753856747115866768095859354146757510519576382887706071653036914256273173891543442157877384439983913724009876651431737714821426505
